AssertionError: wrong number of args to assert

